factory.js

```diff
@@ -113,41 +114,10 @@ Factory.prototype.init = function init(next) {
   // Read the package.json AND the wrhs.toml
   //
   debug(`Read package.json: ${ pkg } and wrhs.toml: ${ wrhs }`);
-  async.parallel([
-    function packJson(fn) {
-      fs.readFile(pkg, 'utf-8', function read(error, content) {
-        if (error) return void fn(error);
-
-        try {
-          factory.pkg = JSON.parse(content);
```

Comment:
do we have everything in the config that this needed

Reply:
looks like it's also used to pull out the browserify stuff:

https://github.com/warehouseai/workers-factory/blob/c3558a8bb16e2600fc8ac380df8f868ac295de5c/workers/browserify.js#L39